client/landing/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
```

Comment:
Are there any places for listing out the team?  Photos and such?  I'm okay if we don't, was more curious than anything else